src/host.cpp

```diff
@@ -1075,6 +1079,27 @@ namespace rhost {
             }
         }
 
+        void setCallbacksWindows(structRstart& rp) {
```

Comment:
This feels like the ifdef should be around the entire function (including the definition), since they don't do anything on other platforms.